### DISCUSSION TITLE
How to I step by step install this please help

### DISCUSSION THREAD

**Discussion by DOTBU** [dyc3/steamguard-cli#210] [Q&A] [answered]
Im using windows

**Comment by dyc3** [comment]
steamguard-cli doesn't have an "install" process. It's intended to be used from the command line. Take a look at the marked answer in #204.

**Comment by DOTBU** [comment]
Tried to run the exe file but now its not showing

**Reply by dyc3** [reply, marked as answer]
By default, Windows does not keep the command prompt window open if you run a command line application from the regular file browser. Please take a look at #204, because it describes exactly how to get it to work on windows.

**Comment by DOTBU** [comment]
thank you but i think this is impossible for me to do tried it but its not working

**Comment by DOTBU** [comment]
Sorry but i dont have the knowledge to follow

**Comment by DOTBU** [comment]
C:\Users\User\Downloads>steamguard -m C:\Users\User\Desktop\SDA\maFiles
INFO - reading manifest from C:\Users\User\Desktop\SDA\maFiles
INFO - Migrating manifest
ERROR - An unexpected error occurred during manifest migration: Failed to load some accounts: [DeserializationFailed(Error("invalid type: null, expected a string", line: 1, column: 764))]

Caused by:
    Failed to load some accounts: [DeserializationFailed(Error("invalid type: null, expected a string", line: 1, column: 764))]

**Reply by DOTBU** [reply]
This is what i get when i try this

**Reply by DOTBU** [reply]
I just want to import my account

**Reply by Szarikow** [reply]
Same issue
